src/Components/Web.Extensions/src/wwwroot/inputFile.js

```diff
@@ -0,0 +1,143 @@
+(function () {
```

Comment:
At the moment, do developers have to reference this manually?

Totally fine if so, but when you do the JSObjectReference work, hopefully we could tweak this so it's an ES6 module and we load it automatically on demand via a call to `import` that returns a JSObjectReference to the module instance.

Obviously that's a separate work item, but just wanted to check if that lines up with your thinking.

Reply:
Developers will have to reference it manually for now, as they do for the HTML head components. That would be great if we could remove that step via your suggestion.